src/qml/editorwidgets/CheckBox.qml

```diff
@@ -1,9 +1,12 @@
-import QtQuick 2.0
-import QtQuick.Controls 2.4
+import QtQuick 2.12
+import QtQuick.Controls 2.12
 
 Item {
   signal valueChanged( var value, bool isNull )
 
+  property alias checked: checkBox.checked
+  property alias indicator: checkBox.indicator
```

Comment:
What are these changes for?

Reply:
they are used outside

Reply:
see https://github.com/opengisch/QField/blob/master/src/qml/editorwidgets/ValueRelation.qml#L108-L117

Reply:
Oh, we are using the CheckBox **editor widget** in here?

Reply:
Is this still required after removing the `.` import?